src/theme/tabs/switcher/demo.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { MemoryRouter } from 'react-router-dom';
-import DemoRenderer from '@basics/demoRenderer';
+import DemoRenderer from '@views/../../demo/demoRenderer';
```

Comment:
@soroushm Import seems incorrect here as it's both an aliased and relative route